### PR TITLE
Add effects for new Howl of the Wild Animal Forms.

### PR DESCRIPTION
**Pull request by arthurtrumpet**
Added the new Animal Forms from Howl of the Wild PDF.
Added to spell-effects:
Spell Effect: Animal Form (Crab)
Spell Effect: Animal Form (Crocodile)
Spell Effect: Animal Form (Orca)
Spell Effect: Animal Form (Seal)
Updated description in spells to link to new effects:
Animal Form

### CODE REVIEW

File: packs/spell-effects/spell-effect-animal-form-crab.json [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
...Melee</strong> <span class=\"action-glyph\">1</span> big claw, <strong>Damage</strong> 2d8 piercing;</li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> little claw (agile), <strong>Damage</strong> 2d4 piercing.</li><li style=\"box-sizing:border-box;user-select:text;scrollbar-width:initial;scrollbar-color:initial\"><span style=\"font-family:Roboto, sans-serif\">can breathe in water and in air.</span></li></ul></li></ul><hr /><p><strong>Heightened (3rd)</strong> You instead gain 10 temporary HP, AC = 17 + your level, attack modifier +14, damage bonus +5, and Athletics +14.</p>\n<p><strong>Heightened (4th)</strong> Your battle form is Large and your attacks have 10-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 15 temporary HP, AC = 18 + your level, attack modifier +16, damage bonus +9, and Athletics +16.</p>\n<p><strong>Heightened (5th)</strong> Your battle form is Huge and your attacks have 15-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 20 temporary HP, AC = 18 + your level, attack modifier +18, damage bonus +7 and double the number of damage dice, and Athletics +20.</p>"
```
Comment (TikaelSol) [inline packs/spell-effects/spell-effect-animal-form-crab.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Animal Form]</p>\n<p>You call upon primal energy to transform yourself into a Medium animal battle form. When you first cast this spell, choose a form.</p>\n<p>You can decide the specific type of animal (such as lion or snow leopard for cat), but this has no effect on the form's Size or statistics. While in this form, you gain the animal trait. You can Dismiss the spell.</p>\n<p>You gain the following statistics and abilities regardless of which battle form you choose:</p><ul><li>AC = 16 + your level. Ignore your armor's check penalty and Speed reduction.</li><li>5 temporary Hit Points.</li><li>Low-light vision and imprecise scent 30 feet.</li><li>One or more unarmed melee attacks specific to the battle form you choose, which are the only attacks you can Strike with. You're trained with them. Your attack modifier is +9, and your damage bonus is +1. These attacks are Strength based (for the purpose of the @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled] condition, for example). If your unarmed attack bonus is higher, you can use it instead.</li><li>Athletics modifier of +9, unless your own modifier is higher.</li></ul><p>You also gain specific abilities based on the type of animal you choose:</p><ul><li><p><strong>Crab</strong></p><ul><li>Speed 25 feet, swim Speed 15 feet;</span></li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> big claw, <strong>Damage</strong> 2d8 piercing;</li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> little claw (agile), <strong>Damage</strong> 2d4 piercing.</li><li>can breathe in water and in air.</span></li></ul></li></ul><hr /><p><strong>Heightened (3rd)</strong> You instead gain 10 temporary HP, AC = 17 + your level, attack modifier +14, damage bonus +5, and Athletics +14.</p>\n<p><strong>Heightened (4th)</strong> Your battle form is Large and your attacks have 10-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 15 temporary HP, AC = 18 + your level, attack modifier +16, damage bonus +9, and Athletics +16.</p>\n<p><strong>Heightened (5th)</strong> Your battle form is Huge and your attacks have 15-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 20 temporary HP, AC = 18 + your level, attack modifier +18, damage bonus +7 and double the number of damage dice, and Athletics +20.</p>"
```
Removing formatting cruft from copy/paste

File: packs/spell-effects/spell-effect-animal-form-crocodile.json [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
...scrollbar-color:initial\"><span style=\"font-family:Roboto, sans-serif\">Speed 25 feet, swim Speed 30 feet;</span></li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> jaws, <strong>Damage</strong> 2d8 piercing;</li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> tail (agile), <strong>Damage</strong> 1d8 bludgeoning.</li><li>can hold your breath for the duration of the transformation.</li></ul></li></ul><hr /><p><strong>Heightened (3rd)</strong> You instead gain 10 temporary HP, AC = 17 + your level, attack modifier +14, damage bonus +5, and Athletics +14.</p>\n<p><strong>Heightened (4th)</strong> Your battle form is Large and your attacks have 10-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 15 temporary HP, AC = 18 + your level, attack modifier +16, damage bonus +9, and Athletics +16.</p>\n<p><strong>Heightened (5th)</strong> Your battle form is Huge and your attacks have 15-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 20 temporary HP, AC = 18 + your level, attack modifier +18, damage bonus +7 and double the number of damage dice, and Athletics +20.</p>"
```
Comment (TikaelSol) [inline packs/spell-effects/spell-effect-animal-form-crocodile.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Animal Form]</p>\n<p>You call upon primal energy to transform yourself into a Medium animal battle form. When you first cast this spell, choose a form.</p>\n<p>You can decide the specific type of animal (such as lion or snow leopard for cat), but this has no effect on the form's Size or statistics. While in this form, you gain the animal trait. You can Dismiss the spell.</p>\n<p>You gain the following statistics and abilities regardless of which battle form you choose:</p><ul><li>AC = 16 + your level. Ignore your armor's check penalty and Speed reduction.</li><li>5 temporary Hit Points.</li><li>Low-light vision and imprecise scent 30 feet.</li><li>One or more unarmed melee attacks specific to the battle form you choose, which are the only attacks you can Strike with. You're trained with them. Your attack modifier is +9, and your damage bonus is +1. These attacks are Strength based (for the purpose of the @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled] condition, for example). If your unarmed attack bonus is higher, you can use it instead.</li><li>Athletics modifier of +9, unless your own modifier is higher.</li></ul><p>You also gain specific abilities based on the type of animal you choose:</p><ul><li><p><strong>Crocodile</strong></p><ul><li>Speed 25 feet, swim Speed 30 feet;</span></li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> jaws, <strong>Damage</strong> 2d8 piercing;</li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> tail (agile), <strong>Damage</strong> 1d8 bludgeoning.</li><li>can hold your breath for the duration of the transformation.</li></ul></li></ul><hr /><p><strong>Heightened (3rd)</strong> You instead gain 10 temporary HP, AC = 17 + your level, attack modifier +14, damage bonus +5, and Athletics +14.</p>\n<p><strong>Heightened (4th)</strong> Your battle form is Large and your attacks have 10-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 15 temporary HP, AC = 18 + your level, attack modifier +16, damage bonus +9, and Athletics +16.</p>\n<p><strong>Heightened (5th)</strong> Your battle form is Huge and your attacks have 15-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 20 temporary HP, AC = 18 + your level, attack modifier +18, damage bonus +7 and double the number of damage dice, and Athletics +20.</p>"
```

File: packs/spell-effects/spell-effect-animal-form-orca.json [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
...f animal you choose:</p><ul><li><p><strong>Orca</strong></p><ul><li style=\"box-sizing:border-box;user-select:text;scrollbar-width:initial;scrollbar-color:initial\"><span style=\"font-family:Roboto, sans-serif\">swim Speed 35 feet;</span></li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> jaws, <strong>Damage</strong> 2d8 piercing;</li><li>can hold your breath for the duration of the transformation.</li></ul></li></ul><hr /><p><strong>Heightened (3rd)</strong> You instead gain 10 temporary HP, AC = 17 + your level, attack modifier +14, damage bonus +5, and Athletics +14.</p>\n<p><strong>Heightened (4th)</strong> Your battle form is Large and your attacks have 10-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 15 temporary HP, AC = 18 + your level, attack modifier +16, damage bonus +9, and Athletics +16.</p>\n<p><strong>Heightened (5th)</strong> Your battle form is Huge and your attacks have 15-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 20 temporary HP, AC = 18 + your level, attack modifier +18, damage bonus +7 and double the number of damage dice, and Athletics +20.</p>"
```
Comment (TikaelSol) [inline packs/spell-effects/spell-effect-animal-form-orca.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Animal Form]</p>\n<p>You call upon primal energy to transform yourself into a Medium animal battle form. When you first cast this spell, choose a form.</p>\n<p>You can decide the specific type of animal (such as lion or snow leopard for cat), but this has no effect on the form's Size or statistics. While in this form, you gain the animal trait. You can Dismiss the spell.</p>\n<p>You gain the following statistics and abilities regardless of which battle form you choose:</p><ul><li>AC = 16 + your level. Ignore your armor's check penalty and Speed reduction.</li><li>5 temporary Hit Points.</li><li>Low-light vision and imprecise scent 30 feet.</li><li>One or more unarmed melee attacks specific to the battle form you choose, which are the only attacks you can Strike with. You're trained with them. Your attack modifier is +9, and your damage bonus is +1. These attacks are Strength based (for the purpose of the @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled] condition, for example). If your unarmed attack bonus is higher, you can use it instead.</li><li>Athletics modifier of +9, unless your own modifier is higher.</li></ul><p>You also gain specific abilities based on the type of animal you choose:</p><ul><li><p><strong>Orca</strong></p><ul><li>swim Speed 35 feet;</span></li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> jaws, <strong>Damage</strong> 2d8 piercing;</li><li>can hold your breath for the duration of the transformation.</li></ul></li></ul><hr /><p><strong>Heightened (3rd)</strong> You instead gain 10 temporary HP, AC = 17 + your level, attack modifier +14, damage bonus +5, and Athletics +14.</p>\n<p><strong>Heightened (4th)</strong> Your battle form is Large and your attacks have 10-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 15 temporary HP, AC = 18 + your level, attack modifier +16, damage bonus +9, and Athletics +16.</p>\n<p><strong>Heightened (5th)</strong> Your battle form is Huge and your attacks have 15-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 20 temporary HP, AC = 18 + your level, attack modifier +18, damage bonus +7 and double the number of damage dice, and Athletics +20.</p>"
```

File: packs/spell-effects/spell-effect-animal-form-seal.json [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
...ul><li><p><strong>Seal</strong></p><ul><li style=\"box-sizing:border-box;user-select:text;scrollbar-width:initial;scrollbar-color:initial\"><span style=\"font-family:Roboto, sans-serif\">Speed 20 feet, swim Speed 30 feet;</span></li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> jaws (grapple), <strong>Damage</strong> 2d6 piercing;</li><li>can hold your breath for the duration of the transformation.</li></ul></li></ul><hr /><p><strong>Heightened (3rd)</strong> You instead gain 10 temporary HP, AC = 17 + your level, attack modifier +14, damage bonus +5, and Athletics +14.</p>\n<p><strong>Heightened (4th)</strong> Your battle form is Large and your attacks have 10-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 15 temporary HP, AC = 18 + your level, attack modifier +16, damage bonus +9, and Athletics +16.</p>\n<p><strong>Heightened (5th)</strong> Your battle form is Huge and your attacks have 15-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 20 temporary HP, AC = 18 + your level, attack modifier +18, damage bonus +7 and double the number of damage dice, and Athletics +20.</p>"
```
Comment (TikaelSol) [inline packs/spell-effects/spell-effect-animal-form-seal.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Animal Form]</p>\n<p>You call upon primal energy to transform yourself into a Medium animal battle form. When you first cast this spell, choose a form.</p>\n<p>You can decide the specific type of animal (such as lion or snow leopard for cat), but this has no effect on the form's Size or statistics. While in this form, you gain the animal trait. You can Dismiss the spell.</p>\n<p>You gain the following statistics and abilities regardless of which battle form you choose:</p><ul><li>AC = 16 + your level. Ignore your armor's check penalty and Speed reduction.</li><li>5 temporary Hit Points.</li><li>Low-light vision and imprecise scent 30 feet.</li><li>One or more unarmed melee attacks specific to the battle form you choose, which are the only attacks you can Strike with. You're trained with them. Your attack modifier is +9, and your damage bonus is +1. These attacks are Strength based (for the purpose of the @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled] condition, for example). If your unarmed attack bonus is higher, you can use it instead.</li><li>Athletics modifier of +9, unless your own modifier is higher.</li></ul><p>You also gain specific abilities based on the type of animal you choose:</p><ul><li><p><strong>Seal</strong></p><ul><li>Speed 20 feet, swim Speed 30 feet;</span></li><li><strong>Melee</strong> <span class=\"action-glyph\">1</span> jaws (grapple), <strong>Damage</strong> 2d6 piercing;</li><li>can hold your breath for the duration of the transformation.</li></ul></li></ul><hr /><p><strong>Heightened (3rd)</strong> You instead gain 10 temporary HP, AC = 17 + your level, attack modifier +14, damage bonus +5, and Athletics +14.</p>\n<p><strong>Heightened (4th)</strong> Your battle form is Large and your attacks have 10-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 15 temporary HP, AC = 18 + your level, attack modifier +16, damage bonus +9, and Athletics +16.</p>\n<p><strong>Heightened (5th)</strong> Your battle form is Huge and your attacks have 15-foot reach. You must have enough space to expand into or the spell is lost. You instead gain 20 temporary HP, AC = 18 + your level, attack modifier +18, damage bonus +7 and double the number of damage dice, and Athletics +20.</p>"
```

File: packs/spell-effects/spell-effect-animal-form-crab.json [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+                                "damageType": "piercing",
+                                "dice": 2,
+                                "die": "d8"
+                            },
+                            "img": "systems/pf2e/icons/unarmed-attacks/pincer.webp",
+                            "traits": [
+                                "unarmed"
+                            ]
+                        },
+                        "little-claw": {
```
Comment (TikaelSol) [inline packs/spell-effects/spell-effect-animal-form-crab.json:54]: We should add this localization to re-en.json

**Review by arthurtrumpet — COMMENTED** [review]
Applied suggestions

**Pull request merged**